vendor/data/codelists/IATI/2_03/activity/sector.yml

```diff
@@ -2237,7 +2227,7 @@ data:
   description: 'Costs incurred in donor countries for basic assistance to asylum seekers
     and refugees from developing countries, up to 12 months: temporary sustenance
     other than food and shelter (code 93011), training (93012) and health (93013),
-    i.e. cash “pocket money” to cover subsistence costs and assistance in the asylum
+    i.e. cash ''pocket money'' to cover subsistence costs and assistance in the asylum
```

Comment:
Would it be preferable to use double-quotes (") for the outer layer of the description, so we can keep the less confusing single single-quotes layer around 'pocket money'?

Reply:
Didn't see that! I wonder if that came from my conversion to yaml from the orginal json, thanks for spotting, I'll have a look and let you know.

Reply:
I jumped on to a rails console and created a Codelist from this file with:

`sectors = Codelist.new(type: "sector").to_objects_with_description`

Then I got the description for this sector:

```
sector = sectors.find { |sector| sector.code == "93014" }
sector.description
```

I get: 

```"Costs incurred in donor countries for basic assistance to asylum seekers and refugees from developing countries, up to 12 months: temporary sustenance other than food and shelter (code 93011), training (93012) and health (93013), i.e. cash 'pocket money' to cover subsistence costs and assistance in the asylum procedure: translation of documents, legal and administrative counselling, interpretation services."```

So I am assuming that this is valid yaml escaping.

Reply:
@CristinaRO my view is that I would like to add these IATI code lists without having to do anything to them. 

I'd like the service to be able to use these vendor data files without anyone having to check over them - I can even imagine the world where theyare loaded into the application by users rather then developers (or read from IATI at regular intervals)

For those reasons I am inclined to keep the quotes as is - happy to talk it through if you feel differently, let me know?

Reply:
I thought you had done the escaping manually, but if this is a correct automated process we can hope it will be future-proof as well.